client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -3,23 +3,45 @@ import {
 	globe as domainsIcon,
 	commentAuthorAvatar as profileIcon,
 } from '@wordpress/icons';
+import { useEffect, useMemo } from 'react';
 import MaterialIcon from 'calypso/components/material-icon';
 import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
 import { navigate } from 'calypso/lib/navigate';
 import { SFTP_PORT, SFTP_URL } from 'calypso/my-sites/hosting/sftp-card';
-import { useSelector } from 'calypso/state';
+import { useDispatch, useSelector } from 'calypso/state';
+import { requestAtomicSftpUsers, requestAtomicSshAccess } from 'calypso/state/hosting/actions';
 import { getAtomicHosting } from 'calypso/state/selectors/get-atomic-hosting';
 import { isCustomDomain, isNotAtomicJetpack } from '../utils';
 
+function useFetchAtomicHosting( atomicSiteIDs: number[] | undefined ) {
+	const dispatch = useDispatch();
+	useEffect( () => {
```

Comment:
I think it would work but there are a couple of things that I am worried about. As you mentioned in the PR description, with this approach, I don't think we can be consistent with the Raycast extension, and having consistency there is probably best for long-term maintenance. 

I **think** with this, we still don't get the SSH users so it might be best in any case to proceed with the API request.

Reply:
Cool, I closed this PR. Making the API request seems like a good solution. We just need to request the SSH users and we can avoid fetching the hosting information.